packages/addons-v5/test/commands/addons/create.js

```diff
@@ -217,6 +217,7 @@ Use heroku addons:docs heroku-db3 to view documentation
       })
 
       it('waits for response and notifies', () => {
+        // eslint-disable-next-line node/no-extraneous-require
```

Comment:
Were these eslint updates supposed to be in this PR?

Reply:
Some of these probably came through when I rebased. I'll double check that they're still valid. Thank you for catching them!